project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -110,6 +126,17 @@ private static User createUser(String email, String nickname) {
     return new User(email, nickname, userID, docHashes, folderIDs);
   }
 
+  public static long createComment(long userID, String data, String date) {
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("userID", userID);
+    commentEntity.setProperty("data", data);
+    commentEntity.setProperty("date", date);
+      
+    getDatastore().put(commentEntity);
+    long commentID = commentEntity.getKey().getId();
+    return commentID;
```

Comment:
out of curiosity why are we returning a commentID in this function? Maybe return a Comment object instead?

Reply:
we're returning the commentID instead of the actual object because for the response we only need the ID to reload the comment